modules/react/modal/stories/examples/WithoutCloseIcon.tsx

```diff
@@ -9,7 +9,7 @@ import {
   useAssistiveHideSiblings,
   useDisableBodyScroll,
 } from '@workday/canvas-kit-react/popup';
-import {DeleteButton} from '@workday/canvas-kit-react/button';
+import {DeleteButton, PrimaryButton} from '@workday/canvas-kit-react/button';
```

Comment:
I'm guessing `PrimaryButton` was used in this example at some point and then later removed? No need for it now since it's not currently being used.
```suggestion
import {DeleteButton} from '@workday/canvas-kit-react/button';
```